infrastructure/cdn-in-a-box/README.md

```diff
@@ -1,44 +1,141 @@
+functionality (caches do not respond to update queues/routing, and origin serves
+static HTTP content - likely subject to change in favor of proper video streaming in
+the future). Other components will follow as well as details on specific parts of the
+implementation.
+
+## Setup
+The containers run on docker, and require Docker (tested v17.05.0-ce) and Docker
+Compose (tested v1.9.0) to build and run. On most 'nix systems these can be installed
+via the distribution's package manager under the names `docker-ce` and
+`docker-compose`, respectively (e.g. `sudo yum install docker-ce`).
+
+Each container (except the origin) requires an `.rpm` file to install the Traffic Control
+component for which it is responsible. You can either download these `*.rpm` files or
+create them yourself by using the [`pkg`](../../pkg) script at the root of the
+repository. Copy the `*.rpm`s without any version/architecture information to their
+respective component directories, such that their filenames are as follows:
+
+* `edge/traffic_ops_ort.rpm`
+* `mid/traffic_ops_ort.rpm`
+* `traffic_monitor/traffic_monitor.rpm`
+* `traffic_ops/traffic_ops.rpm`
+* `traffic_portal/traffic_portal.rpm`
+
+Also ensure that the edge- and mid-tier caches have copies of the
+[`traffic_ops/to-access.sh`](./traffic_ops/to-access) script in their own directories.
+From this directory, you can accomplish this by running:
+
+```bash
+cp -f traffic_ops/to-access.sh edge/to-access.sh && cp -f traffic_ops/to-access.sh mid/to-access.sh
+```
+
+Finally, run the test CDN using the command:
+
+```bash
+docker-compose up --build
+```
+
+## Components
+> The following assumes that the default configuration provided in
+> [`variables.env`](./variables.env) is used.
+
+Once your CDN is running, you should see a cascade of output on your terminal. This is
+typically the output of the build, then setup, and finally logging infrastructure
+(assuming nothing goes wrong). You can now access the various components of the CDN on
+your local machine. For example, opening [`https://localhost`](https://localhost) should
+show you the default UI for interacting with the CDN - Traffic Portal.
 
-To start it, install `docker-ce` and `docker-compose`.  Create an rpm for each
-component and copy into the respective directory with no version number, e.g.
-`traffic_ops/traffic_ops.rpm`, `traffic_portal/traffic_portal.rpm`.
-    
-    cd infrastructure/cdn-in-a-box
-    docker-compose up --build
+> Note: You will likely see a warning about an untrusted or invalid certificate for
+> components that serve over HTTPS (Traffic Ops & Traffic Portal). If you
+> are sure that you are looking at `https://localhost:N` for some integer `N`, these
+> warnings may be safely ignored via the e.g. `Add Exception` button (possibly hidden
+> behind e.g. `Advanced Options`).
+
+### Traffic Ops
+The API and legacy UI for the CDN
+* URLs:
+	* New Golang endpoints: [`https://localhost:6443`](https://localhost:6443)
```

Comment:
Can we reference as "Go" and not "Golang"?   also should note that any endpoints that haven't yet been rewritten in Go will automatically be proxies to the Perl service.